pkg/utils/init_user_test.go

```diff
@@ -0,0 +1,67 @@
+package utils
+
+import (
+	"fmt"
+	datav1alpha1 "github.com/fluid-cloudnative/fluid/api/v1alpha1"
+	"reflect"
+	"testing"
+)
+
+var uid int64 = 1000
+var gid int64 = 1000
+var name = "test-user-1"
```

Comment:
The name uid, gid is likely to be used as global variable inside the package. I suggest to change them to testUID